svf-llvm/tools/AE/ae.cpp

```diff
@@ -653,17 +654,24 @@ int main(int argc, char** argv)
     AndersenWaveDiff* ander = AndersenWaveDiff::createAndersenWaveDiff(pag);
     PTACallGraph* callgraph = ander->getPTACallGraph();
     builder.updateCallGraph(callgraph);
+    pag->getICFG()->updateCallGraph(callgraph);
+    if (Options::SimplifyICFG()) {
+        ICFGSimplification::simplify(pag->getICFG());
+        if (Options::DumpSimplifiedICFG())
```

Comment:
we can remove this option and use -dump-icfg as the name of the output icfg.simplify.dot is different from original icfg.dot